toolchest_client/tools/tool.py

```diff
@@ -75,10 +79,7 @@ def _prepare_inputs(self):
             self.input_files = files_in_path(self.inputs)
             self.num_input_files = len(self.input_files)
 
-        # Sanitize valid paths and list all files in self.inputs.
-        self.input_files = files_in_path(self.inputs)
-
-        self.num_input_files = len(self.input_files)
```

Comment:
@bcai2 Do you know why the block above was reinserted? It has a git annotation of 12/8/2021.

The same logic should be expressed through the `if self.compress_inputs / else` block above, this variant breaks `cellranger_mkfastq`

Reply:
Seems like it was a merge commit gone wrong (my bad!).

Reply:
👍